comp/swerve100/README.md

```diff
@@ -2,4 +2,70 @@
 
 This is the main comp bot repo.
 
-Part of the code here was (somewhat distantly) derived from 254 code, and other parts were inspired by Roadrunner.
\ No newline at end of file
+Part of the code here was (somewhat distantly) derived from 254 code, and other parts were inspired by Roadrunner.
+
+# Getting Started
+
+## Preparing Your Dev Env
+TODO.
+
+The rest of the instructions assume that you have the following setup:
+1. Git `Team100/all24` repo checked out.
+1. Custom WPI VSCode with extensions installed.
+1. `swerve100` workspace opened.
+
+## Building the RoboRIO Code
+Use the WPI extensions to run the `gradlew` build:
+1. Press `CTRL + SHIFT + P` to get the VSCode Command Mode.
```

Comment:
i usually click on the little "W" icon in the upper left corner, which brings up the WPI commands (like build and deploy)